spec.html

```diff
@@ -39904,6 +39904,57 @@ <h1 id="ao-issuperreference">IsSuperReference ( _V_ )</h1>
         1. Return ? ForIn/OfBodyEvaluation(|BindingIdentifier|, |Statement|, _keyResult_, ~enumerate~, ~varBinding~, _labelSet_).
       </emu-alg>
     </emu-annex>
+
+    <emu-annex id="sec-html-document-dot-all-exotic-object">
+      <h1>HTML Document Dot All Exotic Object</h1>
+      <p>An <dfn>HTML document dot all exotic object</dfn> is an exotic object that behaves like undefined in the <emu-xref href="#sec-toboolean">ToBoolean</emu-xref> and <emu-xref href="#sec-abstract-equality-comparison">Abstract Equality Comparison</emu-xref> abstract operations and when used as an operand for the <emu-xref href="#sec-typeof-operator">typeof operator</emu-xref>. HTML document dot all exotic objects have all the ordinary object internal methods and slots.</p>
+      <emu-note>
+        <p>These objects are never created by this specification. However, the <a href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-all"><code>document.all</code> object</a> in web browsers is a host-created HTML document dot all exotic object that exists for web compatibility purposes. There are no other known examples of this type of object.</p>
+      </emu-note>
+
+      <emu-annex id="sec-html-document-dot-all-exotic-object-to-boolean">
+        <h1>Changes to ToBoolean</h1>
+        <p>The result column in <emu-xref href="#table-10"></emu-xref> for an argument type of Object is replaced with the following algorithm:</p>
+        <emu-alg>
+          1. If _argument_ is an HTML document dot all exotic object, return *true*.
+          1. Return *false*.
+        </emu-alg>
+      </emu-annex>
+
+      <emu-annex id="sec-html-document-dot-all-exotic-object-aec">
+        <h1>Changes to Abstract Equality Comparison</h1>
+        <p>The following steps are inserted after step 3 of the <emu-xref href="#sec-abstract-equality-comparison">Abstract Equality Comparison</emu-xref> algorithm:</p>
```

Comment:
Consider labeling steps and referring to them by labels, as this is kind of fragile. However, it is how things have worked so far, so no need to fix if you'd rather keep as-is.